src/tiled/tilesetdocument.h

```diff
@@ -63,6 +62,9 @@ class TilesetDocument : public Document
     FileFormat *writerFormat() const override;
     void setWriterFormat(TilesetFormat *format);
 
+    TilesetFormat* exportFormat() const override;
```

Comment:
Coding style: please put `*` against the function name.

Reply:
Done